### ISSUE TITLE
Word Break / Hyphens in Containers

### ISSUE THREAD

**Issue by PeteyBoogaloo** [NyaaPantsu/nyaa#322]
In the div style of the comments section of any torrent, rather than hyphenating or continuing on the next line with the whole word, words are separated at the end of the width by `word-break break-all`.

```
-webkit-hyphens: auto;
   -moz-hyphens: auto;
    -ms-hyphens: auto;
        hyphens: auto;
```

**Comment by aerojun** [comment]
Let me test it.

**Comment by yiiTT** [comment]
Implemented but needs to get re-implemented on #658 with the browser specific.
